src/DynamoCoreWpf/Utilities/CrashReportTool.cs

```diff
@@ -158,15 +158,11 @@ private static string FindCERToolInInstallLocations()
         /// <summary>
         /// Calls external CER tool (with UI)
         /// </summary>
+        /// <param name="viewModel">The Dynamo view model</param>
         /// <param name="args"></param>
         /// <returns>True if the CER tool process was successfully started. False otherwise</returns>
         internal static bool ShowCrashErrorReportWindow(DynamoViewModel viewModel, CrashErrorReportArgs args)
         {
-            if (DynamoModel.FeatureFlags?.CheckFeatureFlag("CER", false) == false)
```

Comment:
instead of removing the flag check, why don't we just turn on the flag and keep this check until the next release - if something unexpected happens with the new feature then we can turn the flag back off instead of issuing a hotfix.

Reply:
we could do that.
We would need a production feature flag

Reply:
@Amoursol is currently the only "owner" but I think @jasonstratton could signup as well without any soc compliance issues.

Reply:
> @Amoursol is currently the only "owner" but I think @jasonstratton could signup as well without any soc compliance issues.

Would be good for sure to have someone on the team with access! But always happy to help out where I can.